benchmarks/simple.py

```diff
@@ -8,6 +8,7 @@
 SEM = asyncio.Semaphore(20)
```

Comment:
can we remove this semaphore?

Reply:
seems like this is used by the arq benchmark?

Reply:
sounds good